### ISSUE TITLE
Editors window doesn't show correctly in Redmine 4.

### ISSUE THREAD

**Issue by Snowmean** [mrliptontea/PurpleMine2#83]
Editors window doesn't show correctly in Redmine 4.

![screenshot_2018-12-15_23-29-07](https://user-images.githubusercontent.com/1425787/50043039-5ec27080-00c1-11e9-8220-b26454203fe1.png)

**Comment by adautoserpa** [comment]
@Snowmean I could not use the theme in redmine 4. It does not appear in the list of themes. Could you tell me how you did the installation?

**Comment by Snowmean** [comment]
I've upgraded to redmine4 using the latest [official redmine docker image](https://hub.docker.com/_/redmine).

Here is my `docker-compose.yaml` file

```yaml
version: '3'
services:
  redmine:
    container_name: redmine
    image: redmine:latest
    restart: always
    ports:
      - "80:3000"
    volumes:
      - "~/redmine/files:/usr/src/redmine/files"
      - "~/redmine/plugins:/usr/src/redmine/plugins"
      - "~/redmine/configuration.yml:/usr/src/redmine/config/configuration.yml"
      - "~/redmine/themes/PurpleMine2:/usr/src/redmine/public/themes/PurpleMine2"
      - "~/redmine/db/redmine.db:/usr/src/redmine/sqlite/redmine.db"
```

**Comment by gabriel-cardoso** [comment]
Works for me on Redmine 4 but also observed a few minor bugs.

Any plan on upgrading for version 4 of Redmine ?

Keep up the good work :+1:

**Comment by mrliptontea** [comment]
Hi all,

I merged #84 with few tweaks Redmine 4.0 support should be better now.
